### ISSUE TITLE
welcome_to_clojure.clj: link to the file on Github has incorrect href

### ISSUE THREAD

**Issue by jfhector** [BetterThanTomorrow/dram#19]
I've just installed Calva and opened `welcome_to_clojure.clj` in VSCode.

Thanks for excellent getting started docs!

That file has the following text:
```clj
;; NB: This is work in progress...
;; When you fire up the Getting Started REPL the next
;; time, you will be presented with the option to
;; download new files or continue with the files you
;; have. You can also always find the latest version here:
;; https://github.com/BetterThanTomorrow/dram/blob/dev/drams/welcome_to_clojure.clj
```

When I click the link on the last line, I get a 404.

Looking at the Github repository, it appears that the correct URL is:
https://github.com/BetterThanTomorrow/dram/blob/dev/drams/calva_getting_started/welcome_to_clojure.clj

**Comment by PEZ** [comment]
Thanks for reporting! And for fixing!

**Comment by PEZ** [comment]
Closed by #20
